packages/web/graphql/getPatrons.ts

```diff
@@ -114,17 +114,17 @@ export const getPatrons = async (limit = 50): Promise<Array<Patron>> => {
   return patrons;
 };
 
-export const getPSeedPrice = async (): Promise<string | null | undefined> => {
+export const getPSeedPrice = async (): Promise<number> => {
   const { data, error } = await client
     .query<GetPSeedPriceQuery, GetPSeedPriceQueryVariables>(getPSeedPriceQuery)
     .toPromise();
 
-  if (data?.getPSeedInfo == null) {
+  if (data?.getPSeedInfo?.priceUsd == null) {
     if (error) {
       throw error;
```

Comment:
Is `error` here a string?

```suggestion
      throw new Error(error);
```